requirements.txt

```diff
@@ -22,6 +22,7 @@ orjson==3.10.7
 ortools==9.11.4210
 pandas==2.2.3
 psutil==6.0.0
+publicsuffix2==2.20191221
```

Comment:
I'm a bit skeptical about adding a non-actively maintained package.

Reply:
I understand your point, but in this case I'm only using the library for a list of public suffixes. If we want to avoid using a non-actively maintained package, we could also hard-code it, but I don't think that is any better.